src/Sylius/Component/Core/Taxation/Applicator/CompositeOrderTaxesCalculationStrategy.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Taxation\Applicator;
+
+use Sylius\Bundle\SettingsBundle\Model\Settings;
+use Sylius\Component\Addressing\Model\ZoneInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+
+/**
+ * @author Mark McKelvie <mark.mckelvie@reiss.com>
+ */
+class CompositeOrderTaxesCalculationStrategy implements OrderTaxesApplicatorInterface
```

Comment:
ok ignore me, this is at the strategy level... Should a Strategy implement an Applicator interface? hmmm